code/bmpman/bmpman.cpp

```diff
@@ -992,14 +993,16 @@ bool bm_has_alpha_channel(int handle) {
 void bm_init() {
 	int i;
 
-	mprintf(("Size of bitmap info = " SIZE_T_ARG " KB\n", sizeof(bm_bitmaps) / 1024));
-	mprintf(("Size of bitmap extra info = " SIZE_T_ARG " bytes\n", sizeof(bm_extra_info)));
-
 	if (!bm_inited) {
 		bm_inited = 1;
 		atexit(bm_close);
 	}
 
+	bm_bitmaps = new bitmap_entry[MAX_BITMAPS];
+
+	mprintf(("Size of bitmap info = " SIZE_T_ARG " KB\n", sizeof(bm_bitmaps) / 1024));
```

Comment:
Just noticed that this was not changed. `sizeof(bm_bitmaps)` is the size of the pointer now and not of the underlying array. You need to use `sizeof(bm_bitmaps[0]) * MAX_BITMAPS` here.